architecture/service_catalog/index.adoc

```diff
@@ -0,0 +1,191 @@
+wish to use that service broker to make new types of managed services available
+within their cluster.
++
+A resource of this type encapsulates connection details for that service broker
+and specifies the set of services (and variations of those services) to
+{product-title} that should then be made available to users.
++
+.Example `Broker` Resource
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Broker
+metadata:
+  name: BestDataBase
+spec:
+  url: http://bestdatabase.example.com
+----
+
+Service Class::
+Also synonymous with "service" in the context of the service catalog, a _service
+class_ is a type of managed service offered by a particular broker. Each time a
+new broker resource is added to the cluster, the service catalog controller
+connects to the corresponding service broker to obtain a list of service
+offerings. A new `ServiceClass` resource is automatically created for each.
++
+[NOTE]
+====
+{product-title} also has a core concept called
+xref:../../architecture/core_concepts/pods_and_services.adoc#services[services],
+which are separate Kubernetes resources related to internal load balancing.
+These resources are not to be confused with how the term is used in the context
+of the service catalog and OSB API.
+====
++
+.Example `ServiceClass` Resource
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: ServiceClass
+metadata:
+  name: smallDB
+  brokerName: BestDataBase
+  plans: [...]
+----
+
+Service Plan::
+A _service plan_ is one type of variant of a service class. For example, a
+service class may expose a set of plans that offer varying degrees of
+quality-of-services (QoS), each with a different cost associated with it.
+
+Service Instance::
+A _service instance_ is a provisioned instance of a service class. These are
+created by cluster users who wish to make a new, distinct instance of some type
+of managed service to make that available for use by one or more of their
+applications. When a new `Instance` resource is created, the service catalog
+controller connects to the appropriate service broker and instructs it to
+provision the service instance.
++
+.Example `Instance` Resource
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: my_db
+spec:
+  serviceClassName: smallDB
+----
+
+Application::
+The term _application_ refers to the {product-title} deployment artifacts, for
+example pods running in a user's project, that will use a _service instance_.
+
+Credentials::
+_Credentials_ are information needed by an application to communicate with a
+service instance.
+
+Service Binding::
+A _service binding_ is a link between a service instance and an application.
+These are created by cluster users who wish for their applications to reference
+and use a service instance. Upon creation, the service catalog controller
+creates a Kubernetes secret containing connection details and credentials for
+the service instance. Such secrets can be mounted into pods as usual.
```

Comment:
There is also integration with PodPresets, which allow you to express how the secret should be consumed, and in which pods.